sql/plan/insert.go

```diff
@@ -78,8 +95,7 @@ func (p *InsertInto) Execute(ctx *sql.Context) (int, error) {
 		}
 
 		if !found {
-			def, _ := f.Type.Convert(nil)
-			projExprs[i] = expression.NewLiteral(def, f.Type)
+			projExprs[i] = expression.NewLiteral(nil, f.Type)
```

Comment:
This is a behavior change. Before it was effectively getting a default value for each type and inserting it. You're changing it to literal nils, which is very different (but more correct).

You need test cases demonstrating that this is the correct behavior.

Reply:
Before it was getting the _default_, which wasn't quite the same depending on the type. For example, the default for an `int` would be 0, but the default for a `bool` was `nil`. I'm not too sure on how to write a test that this is correct behavior though...

Reply:
Right, you need to write tests that explicitly insert NULL literals into columns of various types, then verify that these columns contain NULL aftewards. AFAIK they don't have any tests like that currently.

Reply:
Ohhhh gotchaaa